### ISSUE TITLE
human data only?

### ISSUE THREAD

**Issue by pdimens** [maiziezhoulab/RegionIndel#3]
Hello,

I see that the `chr_num` parameter excepts only an integer and that the code then expects the contigs to be named `chr_`. Could this not be generalized to all species if `chr_num` was just `chr` as a `string` and the code then used `>chr` (etc.) to pull out the relevant contig name?

**Comment by maiziex** [comment]
Yes, you can. Ensure that the chromosome names in the reference genome FASTA file follow this format.

**Comment by maiziex** [comment]
Yes, you can. Ensure that the chromosome names in the reference genome FASTA file follow this format.


-
Maizie (Xin) Zhou, Ph.D.
Assistant Professor
Biomedical Engineering, Computer Science, and Data Science Institute
Vanderbilt University

5919 Stevenson Center

Nashville, TN  37235

Cell:  336-473-3540

https://lab.vanderbilt.edu/maizie-zhou-lab/
________________________________
From: Pavel V. Dimens ***@***.***>
Sent: Friday, July 5, 2024 8:27 AM
To: maiziezhoulab/RegionIndel ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [maiziezhoulab/RegionIndel] human data only? (Issue #3)


Hello,

I see that the chr_num parameter excepts only an integer and that the code then expects the contigs to be named chr_. Could this not be generalized to all species if chr_num was just chr as a string and the code then used >chr (etc.) to pull out the relevant contig name?

**Comment by pdimens** [comment]
Would it not be possible to modify the logic such that the user inputs the full contig/chromosome name rather than forcing Contig names to be chr_ ?

**Comment by maiziex** [comment]
Unfortunately, some parts of the code logic need to be modified to accommodate this. The easiest approach for now is to modify the contig name. We plan to make this applicable to a general case in a future version.


-
Maizie (Xin) Zhou, Ph.D.
Assistant Professor
Biomedical Engineering, Computer Science, and Data Science Institute
Vanderbilt University

5919 Stevenson Center

Nashville, TN  37235

Cell:  336-473-3540

https://lab.vanderbilt.edu/maizie-zhou-lab/
________________________________
